### PR TITLE
Remove references to etcd2 in v1.13 since support has been removed

### PR DESCRIPTION
**Pull request by spiffxp**
ref: https://github.com/kubernetes/enhancements/issues/622

Link back to the v1.12 version of the etcd3 doc for
the etcd2->etcd3 migration instructions.

I updated the kube-apiserver reference manually,
unsure if that is auto-generated somehow.

The federation-apiserver can still potentially
support etcd2 so I didn't touch that.

There are master/node yaml files that reference
etcd2.service that are likely highly out of date.
I couldn't find any docs that actually reference
these templates so I removed them

/assign @tfogo 
/cc @AishSundar @liggitt

### CODE REVIEW

**Comment by Bradamant3** [comment]
@spiffxp sorry to do this to you (underdocumented "feature" of new l10n process) -- we need separate PRs for the `en` and `ko` directories, please

**Comment by spiffxp** [comment]
/remove-language ko
moving ko changes to followup pr

**Comment by smarterclayton** [comment]
LGTM as well.

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -201,223 +201,29 @@ If the majority of etcd members have permanently failed, the etcd cluster is con
...
-```shell
-curl -X PUT "http://${host}:${port}/v2/keys/_test"
-```
+As of Kubernetes v1.13.0, etcd2 is no longer supported as a storage backend for
+new or existing Kubernetes clusters. The timeline for Kubernetes support for
+etcd2 and etcd3 is as follows: 
+
+- Kubernetes v1.0: etcd2 only
+- Kubernetes v1.5.1: etcd3 support added, new clusters still default to etcd2
+- Kubernetes v1.6.0: new clusters created with `kube-up.sh` default to etcd3
```
Comment (liggitt) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: and `kube-apiserver` defaults to etcd3

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 2]
```diff
@@ -201,223 +201,29 @@ If the majority of etcd members have permanently failed, the etcd cluster is con
...
+to.
+
+There is no pre-existing process for updating etcd, as this is highly dependent
+on how the etcd cluster was deployed and configured, as well as how the
+Kubernetes cluster was deployed and configured. We recommend that you consult
+your cluster provider's documentation to see if there is a predefined
+solution.
+
+If your cluster was created via `kube-up.sh` and is still using etcd2 as its
+storage backend, please consult the [Kubernetes v1.12 etcd cluster upgrade docs](https://v1-12.docs.kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/#upgrading-and-rolling-back-etcd-clusters)
```
Comment (liggitt) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: per-version docs... TIL

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 3]
```diff
@@ -201,223 +201,29 @@ If the majority of etcd members have permanently failed, the etcd cluster is con
...
+- Kubernetes v1.9.0: deprecation of etcd2 storage backend announced
+- Kubernetes v1.13.0: etcd2 storage backend removed
+
+In the unlikely scenario that your Kubernetes cluster is older than v1.13.0 and
+is still using etcd2 as its  storage backed, you MUST migrate to etcd3 before 
+upgrading the cluster to Kubernetes v1.13.0.  Note that both the etcd version
+must be migrated, as well as the version of the etcd API that Kubernetes talks
+to.
+
+There is no pre-existing process for updating etcd, as this is highly dependent
```
Comment (liggitt) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: suggestion: "The process for migrating from etcd2 to etcd3 is highly dependent..."

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 4]
```diff
@@ -201,223 +201,29 @@ If the majority of etcd members have permanently failed, the etcd cluster is con
...
+new or existing Kubernetes clusters. The timeline for Kubernetes support for
+etcd2 and etcd3 is as follows: 
+
+- Kubernetes v1.0: etcd2 only
+- Kubernetes v1.5.1: etcd3 support added, new clusters still default to etcd2
+- Kubernetes v1.6.0: new clusters created with `kube-up.sh` default to etcd3
+- Kubernetes v1.9.0: deprecation of etcd2 storage backend announced
+- Kubernetes v1.13.0: etcd2 storage backend removed
+
+In the unlikely scenario that your Kubernetes cluster is older than v1.13.0 and
```
Comment (liggitt) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: is the thought that the passage of time will make this statement more and more true? The text from the 1.13 release notes might be clearer (you know you are in this situation from the `--storage-backend` flag):

> Before upgrading a v1.12.x kube-apiserver using `--storage-backend=etcd2` to v1.13.x, etcd v2 data MUST be migrated to the v3 storage backend, and kube-apiserver invocations changed to use --storage-backend=etcd3.

Comment (liggitt) [reply content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: the fact that the v1.13.x server will refuse to start with `--storage-backend=etcd2` with a `etcd2 is no longer a supported storage backend` error is useful for pushing upgraders in that situation to look for docs. Maybe include that error message somewhere in this doc to drive searchers here.

**Comment by liggitt** [comment]
a few nits, mostly looks fine

**Comment by Bradamant3** [comment]
@spiffxp can you get liggit's changes in today? (Friday Nov 30)? Thanks!

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md (hunk 4 above)
Comment (spiffxp) [reply content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: Agree this was the wrong tone to take, thanks for confirming, and your wording is much better

**Pull request merged**